.github/workflows/coverage.yml

```
@@ -1,33 +1,39 @@
+      CODESCENE_CLI_SHA256: ${{ vars.CODESCENE_CLI_SHA256 }}
     steps:
       - uses: actions/checkout@v4
-
-      - uses: dtolnay/rust-toolchain@stable
-        with:
-          components: llvm-tools-preview
-
-      - uses: taiki-e/install-action@cargo-llvm-cov@v2
-
+      - name: Setup Rust
+        uses: leynos/shared-actions/.github/actions/setup-rust@${{ vars.SHARED_ACTIONS_SHA }}
```

actionlint (1.7.7)

context "vars" is not allowed here. no context is available here. see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)